cmeutils/tests/test_structure.py

```diff
@@ -22,6 +22,15 @@ def test_gsd_rdf_samename(self, gsdfile_bond):
         assert norm2 == 1
         assert not np.array_equal(rdf_noex, rdf_ex)
 
+    def test_gsd_rdf_pair_order(self, gsdfile_bond):
```

Comment:
This unit test is a perfect example of testing that the function does what we want 🙌